notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>Can change this section to below content:</p><p> </p><p>import time</p><p>while True:</p><p>  feature_view_sync = admin_client.get_feature_view_sync(name = sync_response.feature_view_sync)</p><p>  if feature_view_sync.run_time.end_time.seconds &gt; 0:</p><p>    status = "Succeed" if feature_view_sync.final_status.code == 0 else "Failed"</p><p>    print(f"Sync {status} for {feature_view_sync.name}.")</p><p>    break</p><p>  else:</p><p>    print(f"Sync ongoing, waiting for 30 seconds.")</p><p>    time.sleep(30)</p><p> </p><p>so it waits till sync finishes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>